docker/aeternity_node_mean16.yaml

```
@@ -19,16 +21,17 @@ keys:
   dir: ./keys
 
 chain:
-  persist: true
+  persist: false
   hard_forks:
     "1": 0
     "2": 2
     "3": 4
-    "4": 5
+    "4": 6
```

Iris is protocol v5 and we can activate directly on block with the first block
```suggestion
  hard_forks:
    "1": 0
    "5": 1
```